lua/browse/bookmarks.lua

```diff
@@ -12,14 +12,55 @@ local M = {}
 -- search bookmarks
 M.search_bookmarks = function(config)
   local bookmarks = config["bookmarks"] or {}
+  local aliases = config["aliases"] or {}
```

Comment:
We don't need `aliases` as an explicit variable, as we can add these too in the `bookmarks`.